tests/dragonfly/acl_family_test.py

```diff
@@ -256,16 +247,11 @@ async def test_bad_acl_file(df_local_factory):
 
 
 @pytest.mark.asyncio
-@dfly_args(
-    {
-        "aclfile": os.environ.get(
-            "DRAGONFLY_PATH/ok.acl", os.path.join(SCRIPTS_DIR, "../../build-dbg/ok.acl")
-        ),
-        "port": 1111,
-    }
-)
+@dfly_args({"port": 1111})
 async def test_good_acl_file(df_local_factory):
-    df = df_local_factory.create()
+    path = df_local_factory.params.path
+    acl = os.path.join(os.path.dirname(path), "ok.acl")
+    df = df_local_factory.create(aclfile=acl)
```

Comment:
See how easier it would've been to place them just locally 😝 

And now you use @dfly_args with a random port to move the default instance out of the way

Reply:
> See how easier it would've been to place them just locally

That's exactly what you suggested :rofl:  

> And now you use @dfly_args with a random port to move the default instance out of the way
I will patch this on the next PR, I will make changes anyway so I didn;t bother